### PR TITLE
Feature/game play

### PR DESCRIPTION
**Pull request by hshahwan**
relates #18 
- create two buttons 
- link the buttons with landing page until other layouts are ready.

### CODE REVIEW

**Comment by Oliversw** [comment]
Are you waiting to merge this until everything else is connected or did you want us to review and merge now @hshahwan? Just wondering because of the 'in-progress' label.

In my recent pull request I have linked a lot of the pages so this will be nice to include soon!

**Comment by hshahwan** [comment]
@Oliversw It's okay if you review and merge, I am just want to prepare right pages to connects with it
issue #19

**Review by ali-7 — CHANGES_REQUESTED** [review]
Doesn't needs to be statefull component, make it statless and add props as parameters to inherit the react routes history props, then u can use it. 

Also i've made a button component in my BannerFooter pr wait for it to be merged the use it instead

**Comment by hshahwan** [comment]
@ali-7  I changed to a stateful component to be scalable in the future if we will create private user, it  will be easy to save the situation of the user if he login in or not,
but it's easy to change

**Review by teenie-quaggard — APPROVED** [review]
In terms of whether or not to make this a stateful component, I think we just keep it stateless until we expand in the future. But either way is fine with me, as you are right - it's easy to change.

File: client/src/Component/views/Play/index.js [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
-        <Footer />
-      </div>
-    );
-  }
-}
+import Button from '../../Common/Button';
+import { ButtonWrapper } from './index.style';
 
+const Play = props => {
+  // eslint-disable-next-line react/prop-types
```
Comment (ali-7) [inline client/src/Component/views/Play/index.js]: make PropType validation

**Review by ali-7 — CHANGES_REQUESTED** [review]

**Review by ali-7 — APPROVED** [review]
good girl :joy:

**Pull request merged**